Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -63,8 +63,8 @@ A point representative is implemented as a term `P` of type `Fin 3 → R`, which
 notation `![x, y, z]`. However, `P` is not syntactically equivalent to the expanded vector
 `![P x, P y, P z]`, so the lemmas `fin3_def` and `fin3_def_ext` can be used to convert between the
 two forms. The equivalence of two point representatives `P` and `Q` is implemented as an equivalence
-of orbits of the action of `R`, or equivalently that there is some unit `u` of `R` such that
-`P = u • Q`. However, `u • Q` is not syntactically equal to `![u ^ 2 * Q x, u ^ 3 * Q y, u * Q z]`,
+of orbits of the action of `Rˣ`, or equivalently that there is some unit `u` of `R` such that
+`P = u • Q`. However, `u • Q` is not syntactically equal to `![u² * Q x, u³ * Q y, u * Q z]`,
```

Comment:
I changed this because `²` and `³` are not Lean code.

Reply:
I don't think we have to use Lean code in docstrings, and I very much prefer docstrings readable in both the editor and the online docs, so I sometimes enclose readable pseudocode rather than actual Lean code in backquotes. However I agree in this case it's probably better to use actual Lean code since you're explain a technical Lean issue, but my version could also be made into actual code by [adding an import](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/superscript.20parser/near/365151927).

Reply:
Wow!